lib/routes/byau/xinwen/index.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const typeMap = {
+    news: 3674,
+    dynamic: 3676,
+};
```

Comment:
It would be better to allow users to enter the ID instead of maintaining a fixed mapping. This way, no further updates would be required if users want to subscribe to a non-listed type, such as 3675.

Reply:
fixed